modules/installation-osp-accessing-api-no-floating.adoc

```diff
@@ -3,10 +3,78 @@
 // * installing/installing_openstack/installing-openstack-installer.adoc
 // * installing/installing_openstack/installing-openstack-installer-custom.adoc
 // * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
+// * installing/installing_openstack/installing-openstack-user.adoc
+// * installing/installing_openstack/installing-openstack-user-kuryr.adoc
+
+ifeval::["{context}" == "installing-openstack-installer-custom"]
+:osp-ipi:
+endif::[]
+ifeval::["{context}" == "installing-openstack-installer-kuryr"]
+:osp-kuryr:
+:osp-ipi:
+endif::[]
+ifeval::["{context}" == "installing-openstack-user"]
+:osp-upi:
+endif::[]
+ifeval::["{context}" == "installing-openstack-user-kuryr"]
+:osp-kuryr:
+:osp-upi:
+endif::[]
+ifeval::["{context}" == "installing-openstack-installer-restricted"]
+:osp-ipi:
+:osp-restricted:
+endif::[]
 
 [id="installation-osp-accessing-api-no-floating_{context}"]
-= Enabling access without floating IP addresses
+= Completing installation without floating IP addresses
+
+// FIP _and_ external network, or FIP _or_ external network?
+You can install {product-title} on {rh-openstack-first} without using a floating IP address and external network. In the
+ifdef::osp-ipi[`install-config.yaml`]
+ifdef::osp-upi[`inventory.yaml`]
+file, leave the following
+ifdef::osp-ipi[parameters]
+ifdef::osp-upi[variables]
+blank:
+
+ifdef::osp-ipi[]
+* `platform.openstack.ingressFloatingIP`
+* `platform.openstack.lbFloatingIP`
+* `platform.openstack.externalNetwork`
+endif::osp-ipi[]
+
+ifdef::osp-upi[]
+* `os_api_fip`
+* `os_ingress_fip`
+* `os_external_network`
+endif::osp-upi[]
+
+ifdef::osp-ipi[]
+If you run the installer from a system that cannot reach the {product-title} API due to a lack of floating IP addresses, installation fails. To prevent installation failure in this case, you can use a proxy network or run the installer from a system that is on the same network as your {product-title} machines.
+endif::osp-ipi[]
+
+ifdef::osp-upi[]
+// FIXME: Do we need this `wait-for` note at all?
+If you run the installer with the `wait-for` option from a system that cannot reach the {product-title} API due to a lack of floating IP addresses, installation fails. To prevent installation failure in this case, you can use a proxy network or run the installer from a system that is on the same network as your {product-title} machines.
+endif::osp-upi[]
 
-If you cannot use floating IP addresses, the {product-title} installation might still finish. However, the installation program fails after it times out waiting for API access.
+You must edit the cluster's networking configuration as a post-installation task.
```

Comment:
We do not explain to users how to setup their DNS so that they can reach the api record.

I suggest we add point like you did with point 3 in [Enabling access with floating IP addresses](https://osp-fipless-osdocs1288--ocpdocs.netlify.app/openshift-enterprise/latest/installing/installing_openstack/installing-openstack-installer-custom.html#installation-osp-accessing-api-floating_installing-openstack-installer-custom) except that instead of a Floating IP these points to the IP address of API and Ingress ports respectively.

In which case we do not need anymore to point to the post-installation task.

Reply:
@mandre Like this? https://github.com/openshift/openshift-docs/pull/25216/commits/f2292b28e9143dd05d7022aec986f2f662b51c2c

Reply:
I think using <API_Port IP> instead of <API_IP_address> would be clearer

Reply:
that is what we name the port in openstack

Reply:
Is the note about configuring the networking post install needed? What are the tasks?

Reply:
@mandre There's an item under "Next steps" that links to https://osp-fipless-osdocs1288--ocpdocs.netlify.app/openshift-enterprise/latest/post_installation_configuration/network-configuration.html#installation-osp-configuring-api-floating-ip_post-install-network-configuration

Reply:
@mandre TBH, I'm not sure how much else needs to happen here, or if that post-install document is appropriate.

Reply:
Can this note be deleted? What are the cases where that post-installation document are useful, if any?

Reply:
I think we can remove it. We have given the user enough information here to understand the networking situation enough to be able to handle it as they see fit post install.

Reply:
Done in https://github.com/openshift/openshift-docs/pull/25216/commits/117dc2a0d14db75baf4fa3e07ced284891aeffc7. LMK if this doesn't work. Otherwise, +1 to let me know that it's QEable.